frontend/src/store/modules/user.ts

```diff
@@ -59,13 +59,16 @@ export default {
     },
     async handleUser(
```

Comment:
Now it's not that clear what does this method is intended for. Please consider renaming.